apps/ecommerce/frontend/src/components/App/Dialog/ResourceSelectionDialog/ResourceSelectionDialog.spec.tsx

```diff
@@ -25,7 +25,7 @@ const mockFetchWithSignedRequest = fetchWithSignedRequest as jest.MockedFunction
 
 const { getByText, getByTestId, findByText } = screen;
 
-describe('ResourceSelectionDialog component', () => {
+describe.skip('ResourceSelectionDialog component', () => {
```

Comment:
is this (and the other skips in this PR) because the test is flaky?

Reply:
Yes, i skipped them because it's ecommerce which is no longer really part of our project and flakey tests

Reply:
oh word, good point 💯